lib/ui/widgets/custom_text_form_field.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+
+class CustomTextFormField extends StatefulWidget {
+  final bool includeTextField;
+  final String labelText;
+  final String hintText;
+  final IconData? icon;
+
+  const CustomTextFormField({
+    super.key,
+    this.includeTextField = true,
+    required this.labelText,
+    required this.hintText,
+    this.icon,
+  });
+
+  @override
+  CustomTextFormFieldState createState() => CustomTextFormFieldState();
+}
+
+class CustomTextFormFieldState extends State<CustomTextFormField> {
+  static const EdgeInsets _contentPadding = EdgeInsets.all(15);
+  static const EdgeInsets _paddingBetweenLabelField = EdgeInsets.only(top: 10);
```

Comment:
**suggestion:** We should always use multiple of 4 for layout and multiple of 2 for sizes. 
-> padding of 16

Reply:
done